.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule ".jenkins/env"]
 	path = .jenkins/env
 	url = https://github.com/VulcanClimateModeling/buildenv.git
+[submodule "FV3/call_py_fort"]
+	path = FV3/call_py_fort
+	url = https://github.com/nbren12/call_py_fort.git
```

Comment:
@nbren12 Not sure if it makes sense to import this from an "external" repository. If there is any work done on this repo, in principle we are in a "grey" zone since it should live inside our GitHub organization. Generally, we should not be committing code to our personal repos while on Vulcan/AI2 payroll.

Reply:
The library is stable IMO. The basic APIs haven't changed in about a year. If it requires significant changes, we can explore other contribution angles. There is some precedent for depending using projects maintained on a personal level (e.g. we use spencer's https://github.com/spencerkclark/xpartition) library. We also contribute to open source projects more broadly (e.g. xarray, cftime, gcsfs, etc), so this doesn't seem that different.